roles/commcare_analytics/templates/create_postgres_dump.sh.j2

```diff
@@ -2,7 +2,7 @@
 TODAY=$(date +"%Y_%m_%d")
 NOW=$(date +"%Y_%m_%d %H:%M:%S")
 
-BACKUP_NAME="postgres_backup_${TODAY}"
+BACKUP_NAME="{{ env }}_postgres_backup_${TODAY}"
```

Comment:
environment name is now added to the back up name.
New back up worked fine on production but things didn't work well on staging because the db "superset" added in the list of dbs to be backed up was not present on staging, this is resolved in a later commit.